pkg/config/aliases.go

```diff
@@ -261,3 +261,15 @@ func LoadWithoutSecret() (*model.Warnings, error) {
 func GetProcessAPIAddressPort() (string, error) {
 	return pkgconfigsetup.GetProcessAPIAddressPort(Datadog())
 }
+
+// ChangeLogLevel changes the log level of the Datadog agent.
+// It takes a `level` string representing the desired log level and a `source` model.Source indicating the desired subconfiguration to update.
+// It returns an error if the log level is invalid or if there is an error setting the log level.
+func ChangeLogLevel(level string, source model.Source) error {
```

Comment:
This file contains only aliases. This is used by OTEL. This function should probably go elsewhere.

Reply:
Oh I didn't know, I moved it in its own file (I didn't found a relevant place to move it to). Done in 3bf2ae77434cc6e601b74f33ae125377a7f3ab89.

Reply:
Done in 644d06c67ecfbb5cc7bfa160bb8aaf55f1edb989.